CommonMC/src/FindMCPrimary_module.cc

```diff
@@ -97,9 +97,9 @@ namespace mu2e {
 	}
       }
     }
-    // handle CRY separately: it simulates secondaries and we have to create the primary ourselves.
+    // handle CRY and CORSIKA separately: it simulates secondaries and we have to create the primary ourselves.
     // This should be done in the generator FIXME!
-    if(pgps.size() >0 && pgps.front()->generatorId() == GenId::cosmicCRY){
+    if(pgps.size() >0 && (pgps.front()->generatorId() == GenId::cosmicCRY || pgps.front()->generatorId() == GenId::cosmicCORSIKA )){
```

Comment:
Does it make sense to also include particles from the Daya Bay generator in this test?
 
In GenId there is a function isCosmic that is defined (and out of date) but is not used anywhere in our code base.  If you also want to include Daya Bay, please update the isCosmic function and use it here.

Everyone: if there are other aggregates of GenIds that it makes sense to define.  Please suggest adding the function to GenId